src/codemagic/cli/cli_app.py

```diff
@@ -449,80 +449,11 @@ def execute(
         ).execute(**execute_kwargs)
 
 
-_Fn = TypeVar("_Fn", bound=Callable[..., object])
```

Comment:
This was moved to `src/codemagic/cli/action.py`. Global Python API remains the same as `action` decorator is still accessible as `codemagic.cli.action`.